guide-us-english.tex

```diff
@@ -286,8 +286,10 @@ \subsection{Continuous Improvement and Impediment Removal}
 In order to scale, impediments need to be removed as soon as possible.
 Unresolved impediments scale with the implementation and cripple productivity.
 In Scrum@Scale we crush impediments to guarantee a scale-free architecture.
+Similarly, we also want to scale improvements, to guarantee everyone benefits from them.
+In both cases, the sooner, the better.
```

Comment:
Don't know if this adds that much. I really like your initiative of making things short and concise, this seems to go the other way.